FogCreek/Glitch-Community
# Olive staircase: don't name collections My Stuff

## Links
* https://olive-staircase.glitch.me/
* https://app.zenhub.com/workspaces/community-5d07d1359dee6059a9688176/issues/fogcreek/glitch-community-work/163

## GIF/Screenshots:
![Screen Shot 2019-08-02 at 10 22 21 AM](https://user-images.githubusercontent.com/6620164/62376828-8f033600-b50f-11e9-8ee6-5802daba0b1e.png)
![Screen Shot 2019-08-02 at 10 22 36 AM](https://user-images.githubusercontent.com/6620164/62376829-8f033600-b50f-11e9-944e-ce73831e86b1.png)
![Screen Shot 2019-08-02 at 10 22 55 AM](https://user-images.githubusercontent.com/6620164/62376830-8f033600-b50f-11e9-82e5-5e8e187171b2.png)

## Changes:
* prevents users from naming a collection My Stuff as soon everyone will already have a collection called My Stuff. The next step after this will be converting everyone's old "My Stuff" collections to the new "My Stuff" collection.

## How To Test:
* try to create a new collection or rename a collection to my stuff, you should not be able to.

## Feedback I'm looking for:
- bugs?
